Generals/Code/GameEngine/Source/GameNetwork/GameSpy/Chat.cpp

```diff
@@ -180,10 +196,11 @@ Bool GameSpyInfo::sendChat( UnicodeString message, Bool isAction, GameWindow *pl
 				req.peerRequestType = PeerRequest::PEERREQUEST_MESSAGEPLAYER;
 				TheGameSpyPeerMessageQueue->addRequest(req);
 			}
-
+			s_prevMsg = message;
```

Comment:
This still sets prevMsg even if nothing is sent - eg if names isEmpty, maybe move this line into if (!names.isEmpty())?